jwql/utils/logging_functions.py

```diff
@@ -10,7 +10,7 @@
 Authors
 -------
 
-    - Catherine Martlin 2018
+    - Catherine Martlin 2018, 2019
```

Comment:
@catherine-martlin Actually, could you remove the years altogether here?  We haven't been including years in our other module docstrings, so removing them would be consistent.